tests/PySys/pysysproject.xml

```diff
@@ -1,37 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <pysysproject>
-	<requires-pysys>1.6.1</requires-pysys>
-	<requires-python>3.7.3</requires-python>
-	<property name="appHome" value="${env.PYSYS_APP_HOME}" default="${testRootDir}/.." pathMustExist="true"/>
-    <property name="tebasedir" value="${env.TEBASEDIR}" />
-    <property name="exampledir" value="${env.EXAMPLEDIR}" />
-    <property name="username" value="${env.C8YUSERNAME}" />
-    <property name="c8ypass" value="${env.C8YPASS}" />
-    <property name="tenant" value="${env.C8YTENANT}" />
-    <property name="deviceid" value="${env.C8YDEVICEID}" />
-    <property name="device" value="${env.C8YDEVICE}" />
+	<requires-pysys>
```

Comment:
Your linter seems to be using tabs instead of spaces. You might wanna change that.

Reply:
Hmm, should check my vs code setup